src/gui/EQ/EQPlot.cpp

```diff
@@ -5,6 +5,46 @@
 
 namespace gui::eq
 {
+
+void drawDbText (juce::Graphics& g, const chowdsp::SpectrumPlotBase& plot, float maxGain)
+{
+    const int numValues = 7;
+    const int textWidth = 50;
+    const int textHeight = 14;
```

Comment:
I think it would be cool if the text height were defined relative to the height of the plot? Then once we know the font height we can determine the width from there (see https://docs.juce.com/master/classFont.html#a23539f1990fd39341b84566a413c051d)